### PR TITLE
move from gopkg.in/juju/idmclient.v1 to gopkg.in/CanonicalLtd/candidclient.v1

### PR DESCRIPTION
**Pull request by rogpeppe**
This change is just mechanical renaming.

### CODE REVIEW

File: candidtest/identity.go [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 	"gopkg.in/macaroon-bakery.v2/bakery/checkers"
 	"gopkg.in/macaroon-bakery.v2/bakery/identchecker"
 
-	"gopkg.in/juju/idmclient.v1"
+	"gopkg.in/CanonicalLtd/candidclient.v1"
 )
 
 // identityClient implement identchecker.IdentityClient. This is used because
-// the idmtest server cannot use idmcliemt.Client because that uses the
+// the candidtest server cannot use candidcliemt.Client because that uses the
```
Comment (mhilton) [inline candidtest/identity.go]: * candidclient

File: params/params_test.go [hunk 2]
```diff
@@ -19,9 +19,9 @@ var usernameUnmarshalTests = []struct {
 }{{
 	username: "user",
 }, {
-	username: "admin@idm",
+	username: "admin@candid",
```
Comment (mhilton) [inline params/params_test.go]: I'm not certain these should be changed like this

**Review by mhilton — APPROVED** [review]
LGTM with a couple of thought and if you could fix that typo too that'd be great.

File: candidtest/identity.go (hunk 1 above)
Comment (rogpeppe) [reply candidtest/identity.go]: done

File: params/params_test.go (hunk 2 above)
Comment (rogpeppe) [reply params/params_test.go]: It's a good question. We should talk about that, although it probably doesn't matter for the purposes of username unmarshaling, as "idm" is just an arbitrary string in that context. I've changed it back anyway.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/candidclient/1/

**Comment by rogpeppe** [comment]
:shipit:

**Comment by rogpeppe** [comment]
landing bot not working yet, so merging manually.

**Pull request merged**